extensions/amp-audioburst-audio/validator-amp-audioburst-audio.protoascii

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-audioburst-audio
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-audioburst-audio"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-audioburst-audio>
+  html_format: AMP
+  tag_name: "AMP-AUDIOBURST-AUDIO"
+  requires_extension: "amp-audioburst-audio"
+  attr_lists: "extended-amp-global"
```

Comment:
The validator whitelists attributes that are valid. If you don't add them here the attributes will be invalid. Take a look at how other extensions add attributes like `src` and other values in their `.protoascii` files.